ru/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -66,55 +66,55 @@
      these variables. Also, translate haveSmtp1.prefix and haveSmtp1.suffix as a single sentence, inserting
      text after the "&quot;" entity in haveSmtp1.suffix, if required grammatically.
 -->
-<!ENTITY haveSmtp1.prefix "Будет использоваться уже существующий сервер исходящей почты (SMTP) — «">
-<!ENTITY haveSmtp1.suffix "&quot;, will be used. You can modify outgoing server settings by choosing Account Settings from the Tools menu.">
+<!ENTITY haveSmtp1.prefix "Будет использоваться существующий сервер исходящей почты (SMTP) «">
+<!ENTITY haveSmtp1.suffix "». Чтобы изменить параметры сервера исходящей почты, в меню «Инструменты» выберите «Параметры учетной записи».">
 
 <!-- Entities for Login page -->
 
-<!ENTITY loginTitle.label "Имена пользователей">
-<!ENTITY usernameDesc.label "Введите имя пользователя, данное вам провайдером электронной почты">
+<!ENTITY loginTitle.label "Имена пользователя">
+<!ENTITY usernameDesc.label "Введите имя пользователя сервера входящей почты, полученное от провайдера электронной почты">
 <!-- LOCALIZATION NOTE (usernameExample.label) : do not translate "&quot;jsmith&quot;" in below line -->
-<!ENTITY usernameExample.label "(например, «i_ivanov»).">
-<!ENTITY usernameLabel.label "Имя пользователя:">
+<!ENTITY usernameExample.label "(например, i_ivanov).">
+<!ENTITY usernameLabel.label "Имя пользователя входящей почты:">
 <!ENTITY usernameLabel.accesskey "I">
-<!ENTITY usernameSmtpDesc.label "Введите свое имя пользователя на сервере исходящей почты, данное вам провайдером электронной почты (обычно оно то же, что и имя пользователя на сервере входящей почты).">
-<!ENTITY usernameSmtpLabel.label "Имя пользователя (SMTP-сервер):">
+<!ENTITY usernameSmtpDesc.label "Введите имя пользователя сервера исходящей почты, полученное от провайдера электронной почты (обычно совпадает с именем пользователя входящей почты).">
+<!ENTITY usernameSmtpLabel.label "Имя пользователя исходящей почты:">
 <!ENTITY usernameSmtpLabel.accesskey "S">
 <!-- LOCALIZATION NOTE (haveSmtp2.prefix and haveSmtp2.suffix) : Do not translate "SMTP" and "&quot;" in
      these variables. Also, translate haveSmtp2.prefix and haveSmtp2.suffix as a single sentence, inserting
      text after the "&quot;" entity in haveSmtp2.suffix, if required grammatically.
 -->
-<!ENTITY haveSmtp2.prefix "Будет использоваться уже существующее имя пользователя сервера исходящей почты (SMTP) —  «">
-<!ENTITY haveSmtp2.suffix "&quot;, will be used. You can modify outgoing server settings by choosing Account Settings from the Tools menu.">
+<!ENTITY haveSmtp2.prefix "Будет использоваться уже существующее имя пользователя исходящей почты (SMTP) «">
+<!ENTITY haveSmtp2.suffix "». Чтобы изменить параметры сервера исходящей почты, в меню «Инструменты» выберите «Параметры учетной записи».">
```

Comment:
I am sure. I have changed the straight quotation mark &quot; (appears as ") to angled quotation mark in order to ensure consistency in the whole application. The one kind of quotation marks should be used everywhere and we should decide which one - angled or straight, I would like to discuss it with you later on.